sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/tpcds/queries/ss_max.sql

```diff
@@ -0,0 +1,14 @@
+SELECT
```

Comment:
might be good to keep since it is a decent benchmark for scan & aggregate performance.